apps/website/app/page.tsx

```diff
@@ -461,6 +450,51 @@ export default function Home() {
             </CardContent>
           </Card>
 
+          {/* Blog Section */}
+          <Card id="blog" className="rounded-xl bg-white/50 p-8 shadow-md">
+            <CardHeader>
+              <CardTitle className="mb-8 text-4xl font-bold text-primary">
+                Latest Updates
+              </CardTitle>
+            </CardHeader>
+            <CardContent>
+              <div className="space-y-6">
+                <ul className="space-y-6">
+                  {blogs.map((blog) => (
+                    <li
+                      key={blog.slug}
+                      className="flex items-start justify-between border-b border-gray-200 pb-4"
+                    >
+                      <div className="w-4/5">
+                        <Link
+                          href={`/blog/${blog.slug}`}
+                          className="block text-2xl font-semibold text-blue-600 hover:underline"
+                        >
+                          {blog.title}
+                        </Link>
+                        <p className="mt-2 text-sm italic text-gray-500">
+                          {blog.date}
+                        </p>
+                      </div>
+                      <div className="w-1/5 text-right text-gray-600">
+                        by {blog.author}
+                      </div>
+                    </li>
+                  ))}
+                </ul>
+
+                <div className="mt-6 text-center">
+                  <Link
+                    href="/blog"
+                    className="inline-block rounded-md bg-primary px-4 py-2 text-lg font-semibold text-white transition hover:bg-primary/80"
```

Comment:
The text color on hover feels off.  We don't really have a color schema for this yet, so maybe just add `hover:text-white` for now.